src/components/typography/components/heading.styles.js

```diff
@@ -1,6 +1,7 @@
 import styled from 'styled-components';
 
```

Comment:
@marzzy It's better that we make a distance between importing packages from node_modules and our own files with a new line.